pkg/ocm/helpers.go

```diff
@@ -36,6 +38,8 @@ import (
 	ocmerrors "github.com/openshift-online/ocm-sdk-go/errors"
 )
 
+const ROSADownloadURL = "https://mirror.openshift.com/pub/openshift-v4/clients/rosa"
```

Comment:
This might still be blocked in customer firewalls

Reply:
That's certainly possible but less likely. The entire ROSA CLI is a wrapper to https://api.openshift.com if they're blocking parts of *.openshift.com then it's out of our hands.